src/esportsbot/bot.py

```diff
@@ -34,6 +47,7 @@ async def send_to_log_channel(guild_id, msg):
 
 @client.event
 async def on_ready():
+    # Initialize the reactionMenuDB and pingme role cooldowns, since this can't be done synchronously
```

Comment:
Do these one-line comments work as docstrings?

Reply:
You're right they don't, let me fix that..